applications/nrf5340_audio/README.rst

```diff
@@ -568,56 +570,15 @@ The following table is a complete overview of the test points on the nRF5340 Aud
 nRF5340 Audio configuration files
 =================================
 
-The nRF5340 Audio uses combinations of multiple :file:`.conf` files for different application versions and device types.
-
-The :file:`prj.conf` file is the main configuration file, and it is always included.
-The configuration files for specifying application versions and device types are named using the format *overlay-<version_or_device>.conf*.
-For example, the configuration file for the ``release`` application version is :file:`overlay-release.conf`.
-
-The following configuration file options are available for the nRF5340 Audio development kit:
-
-* ``release`` - Release version of the application with no debugging features.
-* ``debug`` - Debug version of the application.
-  It has the same option settings as the ``release`` configuration, but also enables debug options.
-* ``headset`` - Application configuration for the headset device type.
-* ``gateway`` - Application configuration for the gateway device type.
-
-You can combine the configuration files as follows to obtain one of four different application configurations:
-
-.. list-table::
-    :widths: auto
-    :header-rows: 1
-
-    * - Configuration file/Application configuration
-      - Headset + debug
-      - Gateway + debug
-      - Headset + release
-      - Gateway + release
-    * - ``prj.conf`` (always used)
-      - ✔
-      - ✔
-      - ✔
-      - ✔
-    * - ``overlay-debug.conf``
-      - ✔
-      - ✔
-      -
-      -
-    * - ``overlay-release.conf``
-      -
-      -
-      - ✔
-      - ✔
-    * - Resulting build directory when using the script
-      - ``dev_headset/build_debug``
-      - ``dev_gateway/build_debug``
-      - ``dev_headset/build_release``
-      - ``dev_gateway/build_release``
-
-This means that when you build the application using the ``headset`` device type and the ``release`` application version (third column), the build process includes :file:`prj.conf`,  :file:`overlay-headset.conf`, and :file:`overlay-release.conf` at the time of building the firmware.
-If you are building using the script, the build files are then placed in the :file:`build/headset_release` directory.
-
-See :ref:`nrf53_audio_app_building` for detailed information about selecting the desired combination of configuration files for your build.
+The nRF5340 Audio application uses :file:`Kconfig.defaults` files to change configuration defaults automatically, based on the different application versions and device types.
+
+Only one :file:`.conf` file is included when building.
+The :file:`prj.conf` file is the default configuration file and it implements the debug application version.
+For the release application version, you need to include the :file:`prj_release.conf` configuration file.
+In the release application version no debug features are enabled.
+
```

Comment:
To @greg-fer : include information about _headset and _gateway files and their relation to BT. They handle BT connection and events and funnel the data to the relevant audio modules. The selection happens at compile time with a specific flag (see below).

I'll write a draft for Erik to review.